screens/Register/Register.tsx

```diff
@@ -35,8 +35,6 @@ const registerValidator = (values: RegisterFormValueProps) => {
     password: '',
   };
 
-  const checkedErrors = errors;
-
   // mail validate
   if (!values.email) {
     errors.email = 'Required';
```

Comment:
ตรงอื่นก็แก้เลย
Required >> กรุณาใส่<label value>
Invalid email address >> อีเมลนี้ไม่ถูกต้อง กรุณาตรวจสอบอีกครั้ง

```suggestion
    errors.email = 'กรุณาใส่อีเมล';
```